functions/Restore-DbaDatabase.ps1

```diff
@@ -221,8 +221,30 @@ folder for those file types as defined on the target instance.
     }
     PROCESS
     {
-        
-        foreach ($f in $path)
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
+		else
+		{
+			if ($Path.StartsWith("\\") -eq $false)
+			{
+				# Many internal functions parse using Get-ChildItem. 
+				# We need to use Test-SqlPath and other commands instead
+				# Prevent people from trying 
+				
+				Write-Warning "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly."
+				return
+				
```

Comment:
When you move this into the foreach loop, you _can_ use continue. Here's the equivalent in Stop-Function:
` Stop-Function -Message "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly." -Silent $Silent -Category InvalidArgument -Target $f -Continue`
This will write a warning and continue with the next path (if any) if not in silent mode. In silent mode it will throw an interrupting error and crash the function.
You'd also need to add a `$Silent` switch parameter to the function.